### PR TITLE
hotfix Some parameters are of type String, but should be numeric (#251031)

### PR DESCRIPTION
**Pull request by vrulevskyi**

### CODE REVIEW

File: src/main/resources/com/fortify/plugin/jenkins/FortifyPlugin/config.jelly [hunk 1]
```diff
@@ -291,11 +291,11 @@
             <f:textbox/>
           </f:entry>
           <f:entry field="timeout" title="Timeout" help="/plugin/fortify/help-timeout.html">
```
Comment (hiro0824) [inline src/main/resources/com/fortify/plugin/jenkins/FortifyPlugin/config.jelly:293]: Isn't it necessary to show unit like s or ms?

File: src/main/resources/com/fortify/plugin/jenkins/FortifyPlugin/global.jelly [hunk 2]
```diff
@@ -61,16 +61,16 @@
 
       <f:validateButton title="Test SSC connection" progress="Testing..." method="testConnection" with="url,token,useProxy,proxyUrl,proxyUsername,proxyPassword" />
       <f:entry title="Maximum issues per page" help="/plugin/fortify/help-breakdownPageSize.html">
-        <f:textbox field="breakdownPageSize" id="breakdownPageSize" />
+        <f:number field="breakdownPageSize" id="breakdownPageSize" />
       </f:entry>
       <f:entry title="Connection timeout" help="/plugin/fortify/help-connectionTimeout.html">
```
Comment (hiro0824) [inline src/main/resources/com/fortify/plugin/jenkins/FortifyPlugin/global.jelly:66]: What is the unit for this?

File: src/main/java/com/fortify/plugin/jenkins/FortifyPlugin.java [hunk 3]
```diff
@@ -1842,13 +1842,13 @@ public UploadSSCBlock(String appName, String appVersion) {
...
 		public UploadSSCBlock(String projectName, String projectVersion, String filterSet, String searchCondition,
-							  String timeout, String pollingInterval) {
+							  Integer timeout, Integer pollingInterval) {
 			this.projectName = projectName != null ? projectName.trim() : "";
 			this.projectVersion = projectName != null ? projectVersion.trim() : "";
 			this.filterSet = filterSet != null ? filterSet.trim() : "";
 			this.searchCondition = searchCondition != null ? searchCondition.trim() : "";
-			this.timeout = timeout != null ? timeout.trim() : "";
-			this.pollingInterval = pollingInterval != null ? pollingInterval.trim() : "";
+			this.timeout = timeout;
```
Comment (hiro0824) [inline src/main/java/com/fortify/plugin/jenkins/FortifyPlugin.java:1850]: Question:
Doesn't need to validate negative number?

Comment (vrulevskyi) [reply src/main/java/com/fortify/plugin/jenkins/FortifyPlugin.java:1850]: We have the range validation for this field - https://github.com/jenkinsci/fortify-plugin/blob/master/src/main/java/com/fortify/plugin/jenkins/FortifyPlugin.java#L1077-L1092.

File: src/main/resources/com/fortify/plugin/jenkins/FortifyPlugin/config.jelly (hunk 1 above)
Comment (vrulevskyi) [reply src/main/resources/com/fortify/plugin/jenkins/FortifyPlugin/config.jelly:293]: You can check out setting helper (question mark icon on the right) where the information about the unit of the field is specified.

File: src/main/resources/com/fortify/plugin/jenkins/FortifyPlugin/global.jelly (hunk 2 above)
Comment (vrulevskyi) [reply src/main/resources/com/fortify/plugin/jenkins/FortifyPlugin/global.jelly:66]: You can check out setting helper (question mark icon) where the information about the unit of the field is specified.

**Comment by hiro0824** [comment]
Thank you for the answers.
It looks conflict happened, can you please fix?

**Comment by vrulevskyi** [comment]
@hiro0824 Rebased.

**Review by hiro0824 — APPROVED** [review]

**Pull request merged**